docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,148 @@
+- The cluster or namespace not being monitored correctly - You would not want to deploy into the dark in this case.
+- Unexpected performance degradation after rolling out a new version of your application.
+
+This is the problem domain on which Keptn acts.
+By using Keptn, in combination with your standard deployment tooling or practices,
+you can move from "I guess it's OK" to "I know it's OK".
+Keptn allows you to wrap governance and automated checks around the deployment
+process to ensure that the end-to-end process of deploying is
+healthy and your application is meeting the SLOs you've defined.
+This is done by configuring Keptn to execute [KeptnTasks](../../docs/reference/crd-reference/task.md) and
+[KeptnEvaluations](../../docs/reference/crd-reference/evaluationdefinition.md) before and after the deployment of a workload.
+With these [KeptnTasks](../../docs/reference/crd-reference/task.md), you can implement checks that ensure your
+infrastructure is in a healthy state before a new version of a deployment is rolled out.
+This way, you can avoid the problems mentioned above by:
+
+- Verifying that your cluster has enough resources to run a workload.
+- Checking the reachability of external services that might be required by your workloads.
+- Ensuring that your cluster is monitored by your monitoring provider.
+- Verifying that your monitoring provider did not detect any open problems within your cluster.
+
+After a successful deployment of a workload, Keptn also provides the means to do post-deployment checks.
+These can be crucial in ensuring that the performance of your newly deployed workload meets your expectations,
+before promoting it into the next stage.
+
+## What’s New in Keptn v2?
+
+User feedback to the Keptn project has been clear, and we have listened.
+Keptn v2 brings 3 key new features:
+
+- Non-Blocking Tasks
+- A new "promotion" stage
+- Ability to pass OpenTelemetry contextual information into, and out of, Keptn
+
+### Non-Blocking Tasks
+
+Keptn offers the ability to perform arbitrary tasks and SLO
+evaluations both before a deployment and after a deployment.
+By design, any pre-deployment task or SLO evaluation
+which fails will block the deployment.
+Often, this is the behaviour you want – if a downstream dependency is unavailable or unhealthy,
+you probably don't want to complete the deployment.
+However, for new Keptn users, this behaviour can appear drastic
+and cause deployments to be "pending" without an obvious cause.
+In Keptn v2 this blocking behaviour for pre tasks and evaluations is now configurable.
+When creating the [KeptnConfig](../../docs/reference/crd-reference/config.md) resource,
+set `spec.blockDeployment: [true|false]`.
+The default behaviour is for Keptn to block deployments (i.e. `spec.blockDeployment: true`)
+until the related pre-deployment tasks and
+evaluations defined in the Keptn configuration have passed successfully.
+
+### The Promotion Stage
```

Comment:
```suggestion
### The promotion stage
```
I find it a bit confusing that we sometimes call this feature "promotion stage" and other times call it "multi-stage application delivery".   At the very least, it seems that this paragraph should include the "multi-stage application delivery" string.